numerics/legendre_body.hpp

```diff
@@ -45,6 +48,13 @@ LegendrePolynomial() {
           std::make_index_sequence<degree_ + 1>>::coefficients);
 }
 
+inline double LegendreNormalizationFactor(int degree, int order) {
+  int const n = degree;
+  int const m = order;
+  return Sqrt(Factorial(n - m) * (n + n + 1) * (2 - (m == 0 ? 1 : 0)) /
+              static_cast<double>(Factorial(n + m)));
```

Comment:
(n + n + 1) * (2 - (m == 0 ? 1 : 0)) / static_cast<double>(FallingFactorial(n + m, 2 * m))

We should probably make Factorial and FallingFactorial return `double` to avoid those static_casts.

Reply:
Done.  Logically it's an integer, let's stick to that.